curator-client/src/main/java/org/apache/curator/drivers/OperationTrace.java

```diff
@@ -33,8 +33,10 @@ public class OperationTrace {
 
     private int returnCode = KeeperException.Code.OK.intValue();
     private long latencyMs;
+    private int requestTransactionCount;
     private long requestBytesLength;
     private long responseBytesLength;
+    private int responseChildrenCount;
```

Comment:
Default this to `-1` ?

`0` is a valid count for node with no children. This way, client can choose to not "record" this field in trace event base on the value.